core/src/main/java/com/googlecode/psiprobe/tools/AccessorFactory.java

```diff
@@ -19,11 +19,6 @@
 public class AccessorFactory {
 
   /**
-   * Instantiates a new accessor factory.
-   */
-  private AccessorFactory() {}
```

Comment:
Sonar might not actually be complaining on this since these are not 'static final'.  I'm going to review real quick to see where the breakdown is.  I still should have seen it when looking.